README.md

```diff
@@ -47,7 +47,7 @@ data:
   media_content_id: https://<unique_cloud_id>.ui.nabu.casa/local/sound_files/doorbell.mp3
   media_content_type: music
   extra:
-    volume: 35  # Can be provided as 0-100 or 0.0-1.0
+    volume: 35  # Can be provided as 0-100 or 0.0-0.99. 1 is considered as the lowest value possible and is the same as 0.01
```

Comment:
This seems a bit wordy. I think just changing the upper range on the floats is enough.
```suggestion
    volume: 35  # Can be provided as 0-100 or 0.0-0.99
```